File: Examples/Features/PDFExporting/PageSetup/ConfigureBlackAndWhite.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Reflection;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.PDFExporting.PdfPageSetup
{
    public class ConfigureBlackAndWhite : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet sheet = workbook.Worksheets[0];

            System.IO.Stream stream = this.GetResourceStream("logo.png");
            GrapeCity.Documents.Excel.Drawing.IShape picture = sheet.Shapes.AddPicture(stream, GrapeCity.Documents.Excel.Drawing.ImageType.PNG, 20, 20, 395, 60);

            //Set text font color.
            sheet.Range["A1:D4"].Value = "Font";
            sheet.Range["A1:D4"].Font.Color = Color.Red;

            //Set cell color
            sheet.Range["A7:D10"].Value = "Green";
            sheet.Range["A7:D10"].Interior.Color = Color.Green;

            //Set print black and white.
            sheet.PageSetup.BlackAndWhite = true;
        }

        public override bool SavePdf
        {
            get
            {
                return true;
            }
        }

        public override bool ShowViewer
        {
            get
            {
                return false;
            }
        }

        public override bool IsNew
        {
            get
            {
                return true;
            }
        }
    }
}

File: Examples.VB/Features/PDFExporting/PageSetup/ConfigureBlackAndWhite.vb
Imports System.Drawing

Namespace Features.PDFExporting.PdfPageSetup
    Public Class ConfigureBlackAndWhite
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim sheet As IWorksheet = workbook.Worksheets(0)
            Dim stream As IO.Stream = GetResourceStream("logo.png")
            Dim picture As IShape = sheet.Shapes.AddPicture(stream, ImageType.PNG, 20, 20, 395, 60)

            'Set text font color.
            sheet.Range("A1:D4").Value = "Font"
            sheet.Range("A1:D4").Font.Color = Color.Red

            'Set cell color
            sheet.Range("A7:D10").Value = "Green"
            sheet.Range("A7:D10").Interior.Color = Color.Green

            'Set print black and white.
            sheet.PageSetup.BlackAndWhite = True
        End Sub
        Public Overrides ReadOnly Property SavePdf As Boolean
            Get
                Return True
            End Get
        End Property
        Public Overrides ReadOnly Property ShowViewer As Boolean
            Get
                Return False
            End Get
        End Property
    End Class
End Namespace
